project.godot

```diff
@@ -56,8 +51,7 @@ _global_script_class_icons={
 "Inventory": "",
 "NakamaRequest": "",
 "Ore": "",
-"Player": "",
-"Serializer": ""
```

Comment:
Something got broken here, the Serializer should still be available on the server

Reply:
Fixed